extensions/default/src/commandsModule.ts

```diff
@@ -153,8 +153,8 @@ const commandsModule = ({ servicesManager, commandsManager }) => {
         const restoreProtocol = !!viewportGridStore[storedHanging];
 
         if (
-          protocolId === hpInfo.hangingProtocolId &&
-          useStageIdx === hpInfo.stageIdx &&
+          protocolId === hpInfo.protocolId &&
```

Comment:
I might be totally off base here but there is a `createHpInfo` in ViewportGrid.tsx that returns the previous property names just changed here. Does that function need an update?

Reply:
That function no longer exists - but it did when you made the comment.